addons/web/static/tests/core/py_utils_tests.js

```diff
@@ -894,15 +894,15 @@ QUnit.module('core', function () {
         var result = pyUtils.eval('contexts', [{
             "__ref": "compound_context",
             "__contexts": [
-                {"__ref": "context", "__debug": "{'type':parent.move_type}",
+                {"__ref": "context", "__debug": "{'move_type':parent.move_type}",
```

Comment:
heu ? oopsy

Reply:
ha mais non, c'est vraiment un test dans web qui utilise la compta ?

Reply:
It is just to test an utility function